src/providers/RelayProvider.ts

```diff
@@ -133,6 +137,16 @@ export class RelayProvider implements TLProvider {
     }
   }
 
+  public async getMetaTxStatus(
+    identityAddress: string,
+    metaTransactionHash: string
+  ) {
+    const { MetaTransactionStatus } = await this.fetchEndpoint<any>(
+      `/identities/${identityAddress}/meta-transactions/${metaTransactionHash}`
```

Comment:
here you used `meta-transactions` but did you not use `meta-transaction-status` in the relay. My suggestion would be to use the link here.